packages/@uppy/webcam/src/index.js

```diff
@@ -63,6 +63,9 @@ module.exports = class Webcam extends UIPlugin {
     this.id = this.opts.id || 'Webcam'
     this.type = 'acquirer'
     this.capturedMediaFile = null
+    // enableMirror is used to toggle mirroring, for instance when discarding the video,
+    // while `opts.mirror` is used to remember the initial user setting
+    this.enableMirror = opts?.mirror ?? true
```

Comment:
Shouldn't this be private instead?
```suggestion
    this.#enableMirror = opts?.mirror ?? true
```